src/mmw/js/src/core/setup.js

```diff
@@ -22,6 +22,7 @@ require('../analyze/filters');
 
 require('bootstrap');
 require('bootstrap-select');
+require('bootstrap-table/dist/bootstrap-table.js');
```

Comment:
Were you able to verify if `bootstrap-table/dist/bootstrap-table.js` can be added to the vendor bundle?